verma2021explaining/openreview/appendix.tex

```diff
@@ -0,0 +1,62 @@
+    \label{fig:t2c-Heart}
+\end{figure}
+\subsection{Synthetic data}
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.4\textwidth, height=3cm]{images/synthetic/synth-rep.png}
+    % \hfill
+    \includegraphics[width=0.4\textwidth, height=3cm]{images/synthetic/modified.png}
+    \caption{a) Synthetic data b) Synthetic data with the modification applied. We modify the data from group 1 across the $0^{th}$ dimension by $ax_{1}^{0} + b$. Here a and b are 2.0, 0.60 respectively.}
+    \label{fig:synth_modified}
+\end{figure}
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.4\textwidth, height=7.2cm]{images/synthetic/synthetic_modified.png}
+    % \hfill
+    \includegraphics[width=0.4\textwidth, height=7.2cm]{images/synthetic/synthetic_modified_scaling.png}
+    \caption{We compare the explanations from the TGT algorithm (left) and the TGT with scaling extension algorithm(right) on the modified synthetic data. We can observe that the TGT with scaling extension has better correctness, and is able to identify the scaling we have applied across the first dimension (i.e. k=0). The $\gamma$ for this dimension is 0.87, which means the scaling factor is $e^{\gamma} \approx 2.38$. Moreover, the translation parameters are approximately same in both the variants of the TGT. }
+    \label{fig:synth_modified_change}
+\end{figure}
+\newpage
+\subsection{Probing Classifier and Feature Importance}
+\label{app:dc}
+\begin{figure}
+    \includegraphics[width=0.4\textwidth, height=7.2cm]{images/diabetes/diabetes-0to1.png}
+    % \hfill
+    \includegraphics[width=0.4\textwidth, height=7.2cm]{images/diabetes/diabetes-0to2.png}
+    \includegraphics[width=0.4\textwidth, height=7.2cm]{images/diabetes/diabetes-1to2.png}
+    \caption{Explanations between different groups for the Pima Indians Diabetes Database.}
+    \label{fig:diabetes_0}
+\end{figure}
+\begin{figure}
+\centering
+    \centering
+     \includegraphics[width=0.4\textwidth, height=4cm]{images/diabetes/0_.png}
+     \includegraphics[width=0.4\textwidth, height=4cm]{images/diabetes/1_.png}
+      \includegraphics[width=0.4\textwidth, height=4cm]{images/diabetes/2_.png}
+    \caption{Feature Importance by the binary classifier for the Pima Indians Diabetes Database. a) (top-left): Feature importance for the classifier between groups 0 and 1. b) (top-right): Feature importance for the classifier between groups 0 and 2. c) (bottom-left): Feature importance for the classifier between groups 1 and 2. We note that the classifiers give significant feature importances to the features which correspond to the deltas (refer fig. \ref{fig:diabetes_0}).}
```

Comment:
Please fix the image relative paths (see my other comment)

Reply:
Hi, thanks for the review. Made the changes and the compilation is now successful. We will review the article content to make final commit.